example/lib/periodic_table.dart

```diff
@@ -31,7 +31,8 @@ class PeriodicTableApp extends StatelessWidget {
         builder: (_, __) {
           return LayoutBuilder(builder: (_, constraints) {
             _viewportSize = constraints.biggest;
-            return const SingleChildScrollView(child: PeriodicTableWidget());
+            // ignore: prefer_const_constructors
```

Comment:
This confused me, so I went in and played with it. You may know all this, but for posterity.

The grid and grid items do re-render on layout changes, but because `AtomicElementWidget.build` relies on a `_viewportSize` variable that is updated on each layout, we need those `build` methods to run repeatedly. The dangers of depending on global state.

There's definitely a more elegant way to do this, but it's example code, so whatever.

Mind adding a comment to describe why we have that ignore there?

Reply:
Good point, will be helpful to readers to better understand the example code!